test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -232,6 +232,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.cloud.base_url (string)',
         'xpack.cloud.cname (string)',
         'xpack.cloud.deployment_url (string)',
+        'xpack.cloud.deployments_url (string)',
```

Comment:
Question: Does this URL contain any sensitive info?

Reply:
No this URL is pointing to "/deployments" by default, unless Control plane decides that it changes (which won't happen anytime soon I think)